zk_toolbox/crates/zk_supervisor/src/messages.rs

```diff
@@ -73,6 +73,7 @@ pub(super) const MSG_DATABASE_NEW_MIGRATION_SUCCESS: &str = "Migration created s
 pub(super) const MSG_INTEGRATION_TESTS_ABOUT: &str = "Run integration tests";
 pub(super) const MSG_REVERT_TEST_ABOUT: &str = "Run revert tests";
 pub(super) const MSG_RECOVERY_TEST_ABOUT: &str = "Run recovery tests";
+pub(super) const MSG_UPGRADE_TEST_ABOUT: &str = "Run recovery tests";
```

Comment:
nit: change message

Reply:
Thank you! Addressed in https://github.com/matter-labs/zksync-era/pull/2515/commits/78799acf728d49383e25de636499e6948d57f72b